### PR TITLE
update nav and add container file

### PR DESCRIPTION
**Pull request by laurafitzgerald**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-8019

## What
Adding in nav changes and placeholder files

## Why
To prepare for upcoming docs work

## Verification Steps
1. Follow the steps 1 and 2 [here](https://github.com/aerogear/mobile-docs#publishing) to build it
2. Verify the navigation works
 
## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -3,19 +3,20 @@ include::{partialsdir}/attributes.adoc[]
 * xref:getting-started.adoc[Getting Started]
 ** xref:getting-started.adoc#introduction[Introduction]
 ** xref:getting-started.adoc#setting-up-aerogear-mobile-services-on-openshift[Setting Up]
+** xref:getting-started.adoc#provision-mdc[Provisioning Mobile Deveoper Console]
 ** xref:getting-started.adoc#registering[Registering a Mobile Client]
```
Comment (jhellar) [inline modules/ROOT/nav.adoc]: This could be changed to `Mobile App`.

File: modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -3,19 +3,20 @@ include::{partialsdir}/attributes.adoc[]
...
-* xref:mobile-clients.adoc[Mobile Clients]
-** xref:mobile-clients.adoc#introduction[Introduction]
-** xref:mobile-clients.adoc#registering[Registering a Mobile Client]
-** xref:mobile-clients.adoc#services[Viewing Available Services]
-** xref:mobile-clients.adoc#building[Building Mobile Apps]
-** xref:mobile-clients.adoc#sdk[AeroGear SDK]
 
+* xref:mobile-developer-console.adoc[Mobile Developer Console]
+** xref:mobile-developer-console.adoc#introduction[Introduction]
+** xref:mobile-developer-console.adoc#registering[Registering a Mobile Client]
```
Comment (jhellar) [inline modules/ROOT/nav.adoc]: Should be `Mobile App`

File: modules/ROOT/nav.adoc [hunk 3]
```diff
@@ -3,19 +3,20 @@ include::{partialsdir}/attributes.adoc[]
...
 
-* xref:mobile-clients.adoc[Mobile Clients]
-** xref:mobile-clients.adoc#introduction[Introduction]
-** xref:mobile-clients.adoc#registering[Registering a Mobile Client]
-** xref:mobile-clients.adoc#services[Viewing Available Services]
-** xref:mobile-clients.adoc#building[Building Mobile Apps]
-** xref:mobile-clients.adoc#sdk[AeroGear SDK]
 
+* xref:mobile-developer-console.adoc[Mobile Developer Console]
+** xref:mobile-developer-console.adoc#introduction[Introduction]
```
Comment (jhellar) [inline modules/ROOT/nav.adoc]: In this doc [1] there is also `Provision MDC` section.

[1] https://docs.google.com/document/d/1V79lcYClM2CY9zHtz-F79rrzHiO_b1VLwHc5EHCeyJw/edit

File: modules/ROOT/nav.adoc [hunk 4]
```diff
@@ -3,19 +3,20 @@ include::{partialsdir}/attributes.adoc[]
...
+** xref:mobile-developer-console.adoc#registering[Registering a Mobile Client]
+*** xref:mobile-developer-console.adoc#sdk[Configure SDK]
+** xref:mobile-developer-console.adoc#services[Viewing Available Services]
+** xref:mobile-developer-console.adoc#building[Building Mobile Apps]
```
Comment (jhellar) [inline modules/ROOT/nav.adoc]: Should we remove this section? Builds won't be productized.

**Comment by jhellar** [comment]
Should we also remove while `mobile-ci-cd-service` section under `Mobile Services`?

File: modules/ROOT/nav.adoc (hunk 4 above)
Comment (laurafitzgerald) [reply modules/ROOT/nav.adoc]: @jhellar we'll still support it upstream so I am leaving it in here. We we wont show these docs downstream.

**Comment by laurafitzgerald** [comment]
@jhellar thanks for the review. I pushed the changes. Free for another review?

**Comment by jhellar** [comment]
@laurafitzgerald looks good. Just a question - in this doc [1] `Showcase Apps`, `API Reference`, `Mobile CLI` and `Glossary` sections were removed. Should we remove it here then?

[1] https://docs.google.com/document/d/1V79lcYClM2CY9zHtz-F79rrzHiO_b1VLwHc5EHCeyJw/edit

**Comment by laurafitzgerald** [comment]
@jhellar Good spot. But we also had the same question about the Services one. Initially we didn't paste it there because there was no change to them. I recall Chris pasting the unchanged one to the proposal during a meeting. There was no discussion about either the Services or Showcase sections being removed. A fail to copy and paste error ;)

**Pull request merged**